Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -403,7 +403,7 @@ private bool CanReadAll(string resource)
             case Resource.RECIM:
                 return _recimParticipantService.IsAdmin(user_name);
             case Resource.LOST_AND_FOUND_MISSING_REPORT:
-                    return user_groups.Contains(AuthGroup.LostAndFoundAdmin);
+                    return (user_groups.Contains(AuthGroup.LostAndFoundAdmin) || user_groups.Contains(AuthGroup.LostAndFoundDevelopers));
```

Comment:
Possibly, I think it's probably okay to leave it, since only specific people will have that auth group (AKA us).  That would be a question for Evan down the line, but for now I want to leave it in so we have access to the admin features for testing and training.